addons/udes_warehouse_classification/tests/test_stock_picking.py

```diff
@@ -0,0 +1,95 @@
+from .common import Base
+
+
+class TestWarehouseClassificationPicking(Base):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.test01_messages = {"productDamson": [{"message": "Inbound Alert"}]}
```

Comment:
Is this the expected result for each test? If its test specific add it to the test, please :)